RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -860,9 +860,9 @@ string replaceMonsterCombatString(monster target, boolean inCombat)
 	{
 		return "skill " + $skill[CHEAT CODE: Replace Enemy];
 	}
-	if(item_amount($item[waffle]) > 0 && !haveUsed($item[waffle]) && auto_is_valid($item[waffle]))
+	if(canUse($item[waffle]))
 	{
-		return "item " + $item[waffle];
+		return useItems($item[waffle], $item[none]);
```

Comment:
```suggestion
		return useItem($item[waffle]);
```

`useItem()` has handling for funkslinging, no need to complicate it unnecessarily.

Reply:
Yeah, useItem double uses waffle for some reason. I changed it back to useItem and this happened.
[useItem waffle.txt](https://github.com/loathers/autoscend/files/12686635/useItem.waffle.txt)

Reply:
Could the issue be `none` vs `nothing`?
![image](https://github.com/loathers/autoscend/assets/4781473/e693345a-c2e4-4efd-8efc-68008f18a92b)

Reply:
Does it still double use when using useItem + change useItem to have `none` as the second item?

Reply:
hmm did a quick test by using 2 different custom combat scripts. One for each of the returned value above. Both resulted in only 1 waffle being used. So `none` vs `nothing` doesn't seem to be the issue